.devcontainer/devcontainer.json

```diff
@@ -1,24 +1,10 @@
-// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
-// https://github.com/microsoft/vscode-dev-containers/tree/v0.187.0/containers/javascript-node-mongo
-// Update the VARIANT arg in docker-compose.yml to pick a Node.js version: 10, 12, 14
+// For format details, see https://aka.ms/devcontainer.json.
+
 {
   "name": "Grouparoo Dev",
```

Comment:
when I opened this file from within the workspace, vscode was saying the `build` property is missing, but from the docs it seems like it's only required when using a dockerfile so i think it's ok?

![image](https://user-images.githubusercontent.com/4368928/147954281-5c5bafa8-ac1b-41a9-9673-33d121aa65bd.png)

Reply:
Things seem to work... so... 🤷